packages/components/src/Link/docs/standalone.tsx

```diff
@@ -0,0 +1,18 @@
+import { Card, Content, Footer, Link } from "@hopper-ui/components";
+import { Heading } from "react-aria-components";
+
+export default function Example() {
+    return (
+        <Card padding="core_240" UNSAFE_width="300px">
+            <Heading>
```

Comment:
During you vacations, i remove the "slots" for heading/content/footer in the card, you can juste add a Text for the content instead

```
<Card padding="core_240" UNSAFE_width="300px">
            <Heading>Card component
            </Heading>
            <Text>
                This card component can include a link which is standalone
            </Text>
            <Link isStandalone href="#">Learn more</Link>
        </Card>
```